docs/v2-system-addon/data_events.md

```diff
@@ -1,6 +1,14 @@
 # Metrics we collect
 
-This is an overview of the different kinds of data we collect in Activity Stream experiment. See [data_dictionary.md](data_dictionary.md) for more details for each field.
+By default, the about:newtab and about:home pages in Firefox (the pages you see when you open a new tab and when you start the browser), will send data back to Mozilla servers about usage of these pages.  The intent is to collect data in order to improve the user's experience while using Activity Stream.  Data about your specific browsing behaiour or the sites you visit is **never transmitted to any Mozilla server**.  At any time, it is easy to **turn off** this data collection by [opting out of Firefox telemetry](https://support.mozilla.org/en-US/kb/share-telemetry-data-mozilla-help-improve-firefox?redirectlocale=en-US&redirectslug=how-can-i-help-submitting-performance-data).
```

Comment:
Not sure if we want to remove [both] the /en-US/ references from the URL and let support.mozilla.org handle locale detection and serving...